### DISCUSSION TITLE
QTable - virtualScroll entire page

### DISCUSSION THREAD

**Discussion by vojvodics** [quasarframework/quasar#12471] [Ideas / Proposals] [answered]
I want to be able to implement a table with auto height and use the virtual scroll on the body instead of specifying the `maxHeight` property.

Here is an example of what I want to achieve: https://codesandbox.io/s/romantic-hoover-3dqyw?file=/src/components/BasicVirtualScroll.vue

I was able to implement this in node_modules by adding a `scrollTarget` prop in the `QTable` component and propagating that prop to the `QVirtualScroll` component. Setting the `scrollTarget=body` on the `QTable` seems to be working, and the virtual scroll is working as expected.

Is there any reason why we shouldn't do this? Does the performance worsen if we use window scroll instead of div container with maxHeight?

(Edit: I'm willing to make a proof of concept PR for this feature - the implementation looks rather simple to me)
(Edit2: The sandbox link may break with an error. Just click a refresh button to fix it)
![Screen Shot 2022-02-14 at 17 36 05](https://user-images.githubusercontent.com/20173051/153906135-7f869bd1-972f-4eab-8302-900908d83679.png)

**Comment by danielfazlijevic** [comment]
Any update on this? :) Limiting height on a table and thus having an additional scrollbar is kind of unnecessary.

**Comment by vojvodics** [comment, marked as answer]
#12574 is merged, that should fix it
